edk2toolext/tests/test_web_dependency.py

```diff
@@ -97,7 +124,7 @@ def test_get_internal_path_root_with_subfolders(self):
         second_level_dir_name = "second_dir"
         inner_dir_path = os.path.join(outer_dir, first_level_dir_name)
         self.assertEqual(WebDependency.get_internal_path_root(outer_dir,
-                         os.path.join(first_level_dir_name, second_level_dir_name)), inner_dir_path)
+                                                              os.path.join(first_level_dir_name, second_level_dir_name)), inner_dir_path)
```

Comment:
Is this whitespace change of value?

Reply:
Flake errors

Reply:
We can't have a line over 120 chars, which it used to be longer?